README.md

```diff
@@ -1,2 +1,31 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+
+##basic word similarity guessing pseudocode
```

Comment:
The worksheet asked for pseudocode for the similarity calculation, not for the whole game! You have done more work than was necessary -- be sure to read specifications carefully, as this is an important skill for software developers